### DISCUSSION TITLE
v1.24.0_34-dev

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#491] [Announcements]
## Highlighted

<img src="https://user-images.githubusercontent.com/27055614/185483179-3d6de0b0-2920-4ec0-b3b5-d5a52ecec713.png" height="500" />

Big thanks to @zoodyy for his implementation of background backup on Android. When enabling this option, you can upload the newly taken photo and video while Immich is in the background. Please help us test and give feedback of this feature.

We are looking to implement the same feature on iOS soon.

### Mobile
* Fixed - Dark mode not auto switch on iOS by @alextran1502 in https://github.com/immich-app/immich/pull/475
* Fixed - WebSocket not getting correct data on mobile by @alextran1502 in https://github.com/immich-app/immich/pull/476
* Updated German translation @MalteKiefer in https://github.com/immich-app/immich/pull/478
* [Android only] background backup by @zoodyy in https://github.com/immich-app/immich/pull/382

### Server
* Fixed - Delete Webp thumbnail upon asset deletion @panoti in https://github.com/immich-app/immich/pull/489
* Feature: Added bull queue prefix by @panoti in https://github.com/immich-app/immich/pull/490

### Chores
* Fixed - Spelling mistakes by @ImMaxUK in https://github.com/immich-app/immich/pull/479
* Readme redistribution and reorganization by @r3nor in https://github.com/immich-app/immich/pull/485

## New Contributors
* @ImMaxUK made their first contribution in https://github.com/immich-app/immich/pull/479
* @panoti made their first contribution in https://github.com/immich-app/immich/pull/490
* @r3nor made their first contribution in https://github.com/immich-app/immich/pull/485
## Support

If you find the project helpful and help you in some ways, you can support the project [one time](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502) or [monthly](https://github.com/sponsors/alextran1502) from Github Sponsor 

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

## F-Droid
You can get the app on F-droid by clicking the image below.

[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
    alt="Get it on F-Droid"
    height="80">](https://f-droid.org/packages/app.alextran.immich)

#### You can also download the app from Google Play Store [here](https://play.google.com/store/apps/details?id=app.alextran.immich) 

*The App version might be lagging behind the latest release due to the review process.*

<p align="left">
  <img src="https://user-images.githubusercontent.com/27055614/164953845-4749bccf-8f9d-45e1-ac41-61fe98d6e5b4.png" width="200" title="Google Play Store">
<p/>

#### You can download the app from Apple AppStore [here](https://apps.apple.com/us/app/immich/id1613945652):

<p align="left">
  <img src="https://user-images.githubusercontent.com/27055614/164953841-de6e3f0f-4fd4-4e25-8327-6c3ccc622fc2.png" width="200" title="Apple App Store">
<p/>

*The App version might be lagging behind the latest release due to the review process.*

Cheers! 🎉

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.24.0_34-dev'>v1.24.0_34-dev</a>.</em>

**Comment by ma-karai** [comment]
Yeah, auto-upload function! BEST
